File: examples/java/documentation/src/main/java/com/mycompany/app/myconstructs/Vpc.java
package com.mycompany.app.myconstructs;

import com.hashicorp.cdktf.TerraformStack;
import software.constructs.Construct;

public class Vpc extends TerraformStack {

    public String id;

    public Vpc(Construct scope, String id){
        super(scope, id);

        this.id = id;
    }

    public String getId(){
        return this.id;
    }
}

File: examples/go/documentation/myconstructs/vpc.go
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0

package myconstructs

import (
	"github.com/aws/constructs-go/constructs/v10"
)

type VPC struct {
	Id string
}

func NewVPC(scope constructs.Construct, id string) *VPC {
	// Content is left to the readers imagination
	return &VPC{
		Id: "dummy",
	}
}
